tfhe/src/core_crypto/prelude.rs

```diff
@@ -3,9 +3,7 @@
 //! The TFHE-rs preludes include convenient imports.
 //! Having `tfhe::core_crypto::prelude::*;` should be enough to start using the lib.
 
-pub use super::algorithms::{
-    add_external_product_assign, polynomial_algorithms, slice_algorithms, *,
-};
+pub use super::algorithms::{polynomial_algorithms, slice_algorithms, *};
```

Comment:
Why not only keeping `*`?

Reply:
hmm 🤔 not sure why that change was in my old branch, guess just * would work just as well

Reply:
done